### ISSUE TITLE
Training on ViT-Large

### ISSUE THREAD

**Issue by rstrudel** [facebookresearch/deit#71]
Hi,

First of all, thanks for releasing the code and model weights. While reading the paper I was wondering, is there any reason you did not show any results on ViT-L with your data-efficient approach?
It seems pretty straightforward to try, so I wondered, is it because of optimization issues or there is little to no improvement over ViT-B when trained on ImageNet-1k with your approach ?
In the ViT paper, when pretraining on ImageNet-21k and JFT-300M, ViT-L provides around +1% and +3% respectively over ViT-B which seems non-negligible. Any insight/experiment on it would be super interesting.

Thanks

**Comment by TouvronHugo** [comment]
Hi @rstrudel ,
Thanks for your message,
For the DeiT paper we did not try to train a larger model than the DeiT-B. Nevertheless, we have observed that it is more complex to train deeper models. Indeed, their training is less stable. We have developed different methods in our last paper [Going deeper with Image Transformers](https://arxiv.org/abs/2103.17239) to try to solve this problem. 
Best,
Hugo
